source/org/zfin/database/repository/PostgresRepository.java

```diff
@@ -25,62 +17,29 @@ public class PostgresRepository {
 
     public static List<PostgresSession> getSystemDatabases(DatabaseFormBean formBean) {
         Session session = HibernateUtil.currentSession();
-        Criteria criteria = session.createCriteria(PostgresSession.class);
-        if (formBean.getOrderBy() != null)
-            criteria.addOrder(Order.asc(formBean.getOrderBy()));
-        else
-            criteria.addOrder(Order.asc("dbname"));
-        return criteria.list();
+        String fromPostgresSession = "from PostgresSession order by dbname";
```

Comment:
do we need to preserve the logic that checks the formBean.getOrderBy() ? If not, we could remove the formBean argument.

Reply:
Yes, we want to list the different databases in alphabetical order.